src/mapping/nav2_map_server/include/nav2_map_server/map_reps/occ_grid_server.hpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_MAP_SERVER__MAP_REPS__OCC_GRID_SERVER_HPP_
+#define NAV2_MAP_SERVER__MAP_REPS__OCC_GRID_SERVER_HPP_
+
+#include <memory>
+#include <string>
+
+#include "nav_msgs/msg/occupancy_grid.hpp"
+#include "nav_msgs/msg/map_meta_data.hpp"
+#include "nav_msgs/srv/get_map.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "nav2_map_server/base_map_server.hpp"
+
+enum MapMode
+{
+  TRINARY,
+  SCALE,
+  RAW
+};
+namespace nav2_map_server
+{
+
+class OccGridServer : public BaseMapServer
+{
+public:
+  explicit OccGridServer(rclcpp::Node::SharedPtr node)
+  : node_(node) {}
+
+  OccGridServer(rclcpp::Node::SharedPtr node, std::string file_name);
+
```

Comment:
Why all of the extra whitespace lines here in this file between the method signatures?

Reply:
There's one line of whitespace between method signatures, which I believed was a common practice (in code I've seen) or at least my personal preference. Maybe it looks excessive for now, but when documentation is properly added, I think it will look fine.